input/fsh/geccoplus/p-allergies.fsh

```diff
@@ -0,0 +1,56 @@
+// Author: Thomas Haese
+// Charité - Universitätsmedizin Berlin
+
+Profile: Allergies
+Parent: $gecco-base-condition
+Id: allergies-geccoplus
+Title: "Allergies"
+Description: "Profile for allergies in the context of GECCO+"
+
+* insert napkon-metadata(2021-11-29, #draft, 0.1.0)
+* category 1.. MS
+  * coding MS
+  * coding ^slicing.discriminator.type = #pattern
+  * coding ^slicing.discriminator.path = "$this"
+  * coding ^slicing.rules = #open
+  * coding contains allergicDisposition 1..1 MS
+  * coding[allergicDisposition] = $sct#609328004 "Allergic disposition (finding)"
+  * coding[allergicDisposition].system 1.. MS
+  * coding[allergicDisposition].code 1.. MS
+* code.coding[icd10-gm] from AllergiesICD (required)
+* code.coding[sct] from AllergiesSCT (required)
+* code.text = "String"
```

Comment:
Die Zeile (* code.text = "String") einfach entfernen. Du kannst noch folgende Zeile einfügen, um prüfen zu lassen, dass code.text ausgefüllt ist, wenn code.coding = snomed "Other" ist. Dazu aber sicher gehen, dass du den snomed code für "other" auch in AllergiesSCT hast (habe ich jetzt nicht überprüft).
```suggestion
* code obeys codeable-concept-text-present-if-code-other
```

Reply:
Ui, das hätte ich nicht gedacht, das da gar nichts hinkommt.